ambry-api/src/main/java/com/github/ambry/config/QuotaConfig.java

```diff
@@ -33,13 +33,14 @@
       QUOTA_CONFIG_PREFIX + "request.enforcer.source.pair.info.json";
   public static final String QUOTA_MANAGER_FACTORY = QUOTA_CONFIG_PREFIX + "manager.factory";
   public static final String QUOTA_ACCOUNTING_UNIT = QUOTA_CONFIG_PREFIX + "accounting.unit";
+  public static final String REJECT_ON_CU_QUOTA_BREACH = QUOTA_CONFIG_PREFIX + "reject.on.cu.quota.breach";
```

Comment:
This is a refactor only PR with no logic change, other than the fact that the instantiation of QuotaChargeCallback is now being controlled by this new config.